src/components/molecules/CalendarBodyDates/CalendarBodyDates.tsx

```diff
@@ -0,0 +1,52 @@
+import { Calendar, GetDatePropsOptions } from 'dayzed'
+import { Box, Button, Text, Wrap, WrapItem } from '@chakra-ui/react'
+import { FC } from 'react'
+import { WEEKDAY_NAMES_SHORT } from '@/@types/Calendar'
+
+interface CalendarBodyDatesProps {
+  calendar: Calendar
+  getDateProps: (data: GetDatePropsOptions) => Record<string, any>
+}
+
+const CalendarBodyDates: FC<CalendarBodyDatesProps> = (props) => {
+  const { calendar, getDateProps } = props
+
+  return (
+    <Box>
+      <Wrap spacing={0}>
+        {WEEKDAY_NAMES_SHORT.map((weekday) => (
+          <WrapItem key={`${calendar.month}${calendar.year}${weekday}`} boxSize={`48px`} justifyContent={`center`}>
+            <Text textAlign={`center`} fontFamily={`body`} fontWeight={`medium`} margin={`auto`}>
+              {weekday[0]}
+            </Text>
+          </WrapItem>
+        ))}
```

Comment:
You can map over a list and genearte a list of react components outside of the return. Would be cleaner. Have some examples online.

Reply:
have extracted the generation of the React components out